drivers/isl29020/isl29020.c

```diff
@@ -59,16 +62,16 @@ int isl29020_read(const isl29020_t *dev)
     uint16_t res;
     int ret;
 
-    i2c_acquire(dev->i2c);
+    i2c_acquire(DEV_I2C);
     /* read lighting value */
-    ret = i2c_read_reg(dev->i2c, dev->address, ISL29020_REG_LDATA, &low);
-    ret += i2c_read_reg(dev->i2c, dev->address, ISL29020_REG_HDATA, &high);
-    i2c_release(dev->i2c);
+    ret = i2c_read_reg(DEV_I2C, DEV_ADDR, ISL29020_REG_LDATA, &low, 0);
+    ret += i2c_read_reg(DEV_I2C, DEV_ADDR, ISL29020_REG_HDATA, &high, 0);
+    i2c_release(DEV_I2C);
     if (ret < 2) {
         return -1;
     }
     res = (high << 8) | low;
-    DEBUG("ISL29020: Raw value: %i - high: %i, low: %i\n", res, high, low);
+    DEBUG("ISL29020: Raw value: %i - high: %i, low: %i\n", res, high, low, 0);
```

Comment:
I guess there is some wrong copy&paste here

Reply:
fixed